packages/@glimmer/core/src/owner.ts

```diff
@@ -0,0 +1,27 @@
+import { DEBUG } from '@glimmer/env';
+
+const OWNER_MAP = new WeakMap<{}, unknown>();
+
+export const OWNER_KEY = `__OWNER_${Math.floor(Math.random() * Date.now())}__`;
```

Comment:
Can we use a "real" symbol for this? Glimmer.js doesn't support IE11 (glimmer-vm does), so I think we can use `Symbol` natively here...

Reply:
We can't use a real one unfortunately, this is for DynamicScope which only currently works with strings (at least, according to types)

Reply:
😭 ok